CHANGELOG.md

```diff
@@ -73,8 +73,8 @@ Read more about migrating a plugin from Piwik 2.X to Piwik 3 on our [Migration g
  * `Report.addReports` lets you add dynamically created reports
  * `Report.filterReports` lets you filter any report
  * `Updater.componentUpdated` triggered after core or a plugin has been updated
- * `PluginManager.pluginInstalled` triggered after a plugin was installed
- * `PluginManager.pluginUninstalled` triggered after a plugin was uninstalled
+ * `Updater.componentInstalled` triggered after a plugin was installed
+ * `Updater.componentUninstalled` triggered after a plugin was uninstalled
```

Comment:
We are using those events in `CustomPiwikJS` and it needs to be updated there if we rename it. Not sure if we should actually rename it as it is now inconsistent with `PluginManager.pluginActivated` and `PluginManager.pluginDeactivated`. Maybe we could have just both events as there might be different use cases where you only want to listen to `pluginActivated` but don't care so much when a dimension component was updated etc? Also it would be maybe more clear for developers to have both events as it would not be so much obvious that you need to listen to an `Updater` event when a plugin was installed. Any thoughts on this @sgiehl @mattab ?